### DISCUSSION TITLE
Асинхронная загрузка таблицы при первом отображении

### DISCUSSION THREAD

**Discussion by ArtemYurov** [moonshine-software/moonshine#1280] [Ideas]
https://github.com/moonshine-software/doc/blob/3.x/ru/components/table-builder.md#async-loading

Метод async() настраивает асинхронную загрузку таблицы:

Этот текст можно понять так, что асинхронная загрузка будет при первом отображении таблицы на странице (чего, к сожалению, не происходит). Поэтому интересно иметь такую возможность, это полезно при долгой загрузке данных через API.

**Comment by lee-to** [comment]
идея мне нравится, подумаю как можно реализовать в ближайшее время
